stl/inc/format

```diff
@@ -1776,7 +1776,7 @@ struct _Format_arg_traits {
         -> basic_string_view<_Char_type>; // not defined
 
     template <class _Traits, class _Alloc>
-    static auto _Phony_basic_format_arg_constructor(const basic_string<_Char_type, _Traits, _Alloc>&)
+    static auto _Phony_basic_format_arg_constructor(basic_string<_Char_type, _Traits, _Alloc>)

```

Comment:
1. I think I see why this change is desirable (`const basic_string&` is a worse match than `_Ty&&` for a modifiable `basic_string`). However, this no longer follows WG21-N4910 [[format.arg]/10](https://eel.is/c++draft/format.arg#10). Does this mean that we need to file an LWG issue?

Reply:
I'd love it if we could address these issues with `basic_format_arg`'s constructor overloads, revise everything to be lvalue-only, and submit an LWG issue. P2418 was very rushed, and did some damage to the wording.